src/js/common/schemaform/SchemaForm.jsx

```diff
@@ -90,11 +90,13 @@ class SchemaForm extends React.Component {
       reviewMode,
       children,
       onSubmit,
-      onChange
+      onChange,
+      safeRenderCompletion
     } = this.props;
     return (
       <div>
         <Form
+            safeRenderCompletion={safeRenderCompletion}
```

Comment:
This is something built into rjsf's Form component that removes some of the async state setting it normally does, which makes it easier to test.